src/reducers/account/index.js

```diff
@@ -92,7 +107,7 @@ const recoverCodeReducer = handleActions({
 const accessKeys = handleActions({
     [getAccessKeys]: (state, { error, payload }) => ({
         ...state,
-        authorizedApps: payload && payload.filter(it => it.access_key && it.access_key.permission.FunctionCall),
+        authorizedApps: payload && payload.filter(it => it.access_key && it.access_key.permission.FunctionCall && it.access_key.permission.FunctionCall.receiver_id !== state.accountId),
```

Comment:
I think we shouldn't hide keys to self for now, as then there is no way to see / remove them at all in wallet.

Reply:
@kcole16 thoughts?